src/C_hw5.2_sortingStation/C_hw5.2_sortingStation.c

```diff
@@ -0,0 +1,118 @@
+#include <stdio.h>
+#include <string.h>
+#include "stack.h"
+
+
+void priority(char op1, char op2, int *n1, int *n2)   //функция для определения приоритетов операторов, где оп1 это оператор из стека, а оп2 оператор из строки
+{
+    if ((op1 == '*') || (op1 == '/')) {
+	*n1 = 2;
+    }
+    else {
+	*n1 = 1;
+    }
+    if ((op2 == '*') || (op2 == '/')) {
+	*n2 = 2;
+    }
+    else {
+	*n2 = 1;
+    }
+}
+
+
+
+int main(void)
+{
+    char infix[] = "(1 + 1) * 2 + (3 + 3) * 4";
+    printf("Инфиксная запись: %s\n\n", infix);
+    int len = strlen(infix);
+    int size = 0;
+    for (int i = 0; i < len; ++i) {                                //считаем сколько в строке операторов и цифр
+	if (infix[i] != '(' && infix[i] != ')' && infix[i] != ' ')
+	    ++size;
+    }
+    char postfix[size];                                          //создаем массив размером сколько посчитали в цикле
+    Block *top = newStack();
+    int tracker = 0;           //индекс свободной ячейки выходного массива
+    char buff;
+
+    for (int i = 0; i < len; ++i) {                //пробегаем по строке
+	switch (infix[i]) {
+	    case '+': case '/': case '-': case '*':
```

Comment:
В стайлгайде ничего про это не сказано, но я первый раз вижу такое форматирование.
Обычно делают так:

```suggestion
	    case '+':
	    case '/':
	    case '-':
	    case '*':
```